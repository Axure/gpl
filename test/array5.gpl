void main()
{
	string[3][20] str;
	char[3][4] ch;
	int[4][3] num;
	int i;
	print(str.len());
	for (i = 0; i < 3; i+=1)
		print(str[i].len());
	print(ch.len());
	for (i = 0; i < 3; i+=1)
		print((ch[i]).len());
	print(num.len());
	for (i = 0; i < 4; i+=1)
		print(num[i].len());
}