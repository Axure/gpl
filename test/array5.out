3
20
20
20
3
4
4
4
4
3
3
3
3
